contracts/StablecoinConverter.sol

```diff
@@ -149,7 +149,8 @@ contract StablecoinConverter is EpochTokenLocker {
         TradeData[] trades;
         uint16[] tokenIdsForPrice;
         address solutionSubmitter;
-        int256 currentFeeCollected;
+        uint256 currentFeeReward;
```

Comment:
nit: I find the `current` part of the fields a bit "doppelt-gemoppelt" (since it's already on the current best solution). Should it be `objectiveValue` and `rewardedFee`?